CHANGELOG.cdf-tk.md

```diff
@@ -17,13 +17,19 @@ Changes are grouped as follows:
 
 ## TBD
 
-## Improved
+### Improved
 
 - When running `cdf-tk auth verify`, if the client does not have access to the `CDF_PROJECT` the user will now get
   a more informative error message.
 - When running `cdf-tk auth verify` and missing the `FunctionAcl(READ)` capability, the user will now get a more
   informative error message when checking the function service status
 
+### Added
+
+- Preview feature `MODULES_CMD` to allow interactive init and automatic upgrade of modules. Activate by running
+  `cdf-tk features set MODULES_CMD --enable`, and deactivate by running `cdf-tk features set MODULES_CMD --disable`.
+  Run `cdf-tk modules init/upgrade` to interactively initialize or upgrade modules.
```

Comment:
Up for discussion, but i think we should log preview features in the changelog. It will be useful to know when they are available from. In addition, that enables us to do patch releases even if there have been no bugfixes for new features.